source/stylesheets/scss/furatto/_forms.scss

```diff
@@ -0,0 +1,336 @@
+      width: 100%;
+    }
+  }
+
+  &.block {
+    width: 100%;
+  }
+
+  &.success,
+  &.error {
+    margin-bottom: 0; 
+  }
+
+  &.success { 
+    border-color: $form-input-success-border-color; 
+    color: $form-label-success-color;
+  }
+  &.error { 
+    border-color: $form-input-error-border-color; 
+    color: $form-label-error-color;
+  }
+}
+
+textarea {
+  height: auto;
+}
+
+select {
+  @include border-radius(0);
+  width: 100%;
+  display: block;
+  background: $form-select-background-color
+  url('data:image/svg+xml;base64, PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHZlcnNpb249IjEuMSIgeD0iMHB4IiB5PSIwcHgiIHdpZHRoPSI2cHgiIGhlaWdodD0iM3B4IiB2aWV3Qm94PSIwIDAgNiAzIiBlbmFibGUtYmFja2dyb3VuZD0ibmV3IDAgMCA2IDMiIHhtbDpzcGFjZT0icHJlc2VydmUiPjxwb2x5Z29uIHBvaW50cz0iNS45OTIsMCAyLjk5MiwzIC0wLjAwOCwwICIvPjwvc3ZnPg==') no-repeat $form-select-background-x-position center;
+  ; 
+
+  height: $form-input-height;
+
+  //Firefox fix
+  text-indent: 0.01px;
+  text-overflow: "";
+  @include appearance(none);
+
+  padding: $form-select-padding;
+  border: $form-select-border-width $form-select-border-style $form-select-border-color;
+  margin-bottom: $form-input-margin-bottom;
+  outline: none;
+  color: $form-select-color;
+  font-size: $form-select-font-size;
+
+  @if $include-form-select-transition {
+    @include single-transition(all $form-select-transition-time $form-select-transition-style);
+  }
+
+  &.radius { @include border-radius($form-select-radius); }
+
+  &.shadow{ @include box-shadow($form-select-box-shadow);
+            border: none; }
+
+  &:hover {
+    background: $form-select-hover-background
+    url('data:image/svg+xml;base64, PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHZlcnNpb249IjEuMSIgeD0iMHB4IiB5PSIwcHgiIHdpZHRoPSI2cHgiIGhlaWdodD0iM3B4IiB2aWV3Qm94PSIwIDAgNiAzIiBlbmFibGUtYmFja2dyb3VuZD0ibmV3IDAgMCA2IDMiIHhtbDpzcGFjZT0icHJlc2VydmUiPjxwb2x5Z29uIHBvaW50cz0iNS45OTIsMCAyLjk5MiwzIC0wLjAwOCwwICIvPjwvc3ZnPg==') no-repeat $form-select-background-x-position center; 
+  }
+}
+
+input[type="checkbox"],
+input[type="radio"],
+input[type="file"] {
+  margin-bottom: $form-input-margin-bottom; 
+}
+
+//Radio buttons and Checkbox style
+
+input[type=radio], input[type=checkbox] {
+    display: none;
+
+    & + label{
+      cursor: pointer; 
+      display: inline-block;
+      font-size: inherit;
+      margin-left: $form-inline-label-margin-left;
+      margin-right: 15px;
+      padding-left: 25px;
+      position: relative;
+      vertical-align: baseline;
+
+      &:before{
+        background-color: #fff;
+        border: 2px solid $radio-button-checkbox-border-color;
+        bottom: 0px;
+        content: "";
+        display: inline-block;
+        height: 20px; 
+        left: 0;   
+        margin-right: 10px;
+        position: absolute;
+        width: 20px;
+      }
+    }
+    &:checked + label:before{
+      color: $radio-button-checkbox-checked-color;
+    }
+
+}
+
+
+input[type=radio]{
+  & + label:before{
```

Comment:
Opening curly brace `{` should be preceded by one space<br>Unnecessary parent selector (&)